pkg/constants/constants.go

```diff
@@ -34,4 +34,6 @@ const (
 	// that do not specify any priority class and there is no priority class
 	// marked as default.
 	DefaultPriority = 0
+
+	PreemptionAdmissionCheckName = "kueue-preemption"
```

Comment:
Why do we need this?

Reply:
I the name of the check, managed by the new controller.

Reply:
This is an exported const, should have a description and some explanation why is it needed.